examples/default/main.tf

```diff
@@ -13,7 +13,7 @@ terraform {
 }
 
 provider "aws" {
-  version             = "1.56.0"
+  version             = ">= 2.20.0"
```

Comment:
Strictly speaking, you can just put this down as `>= 2.20` and `>= 0.12` (don't need the patch version in there). Nitpicks 😅